### PR TITLE
parameterize name

### PR DESCRIPTION
**Pull request by gabemontero**
@bparees Fixes https://github.com/openshift/origin/issues/3571

### CODE REVIEW

**Comment by gabemontero** [comment]
pushed updates based on comments from cakephp-ex

File: openshift/templates/dancer-mysql.json [hunk 1]
```diff
@@ -328,6 +328,13 @@
   ],
   "parameters": [
     {
+      "name": "NAME",
+      "displayName": "Name",
+      "description": "The name assigned to all of the objects defined in this template.",
```
Comment (bparees) [inline openshift/templates/dancer-mysql.json]: frontend

**Comment by gabemontero** [comment]
add of frontend to mysql NAME descr pushed

**Comment by bparees** [comment]
lgtm

**Pull request merged**